src/lib/utilities.js

```diff
@@ -205,6 +206,10 @@ module.exports = {
 				line = path.join(basepath, line);
 			}
 			found = glob.sync(line, { nodir: true, follow: !!followSymlinks });
+			if (os.platform() === 'win32') {
+				// because glob pattern is always in posix format
+				found = found.map(file => path.win32.normalize(file));
+			}
```

Comment:
Apparently, `glob.sync` only works with POSIX formats, which caused it to return POSIX-formatted file paths even on Windows systems. There is a `--posix: false` that theoritically should work around this problem but that option is not working as expected. This has not been a problem in our codebase because the file paths were later processed by `path.relative` [in cloud.js](https://github.com/particle-iot/particle-cli/blob/e8354b92ad54a98637ac2a54caf7d5b9c43ec833/src/cmd/cloud.js#LL823C4-L823C4), which handled the necessary adjustments for the specific operating system but when I tried to write unit tests for that specific code, it started to make the tests fail.

I believe it's best to modify the behavior of glob.sync to generate Windows-formatted paths directly at the source. This way, we can eliminate the need for the caller to handle this issue.